source/agora/network/Manager.d

```diff
@@ -949,16 +949,19 @@ public class NetworkManager
                             {
                                 try
                                 {
-                                    log.trace("getMissingBlockSigs: updating header signature: {} validators: {}", header.signature, header.validators);
-                                    acceptHeader(header);
+                                    auto res = acceptHeader(header);
+                                    if (res is null)
```

Comment:
Do we want to log the error, if any ? Possibly at a low level (trace) since it's data supplied by an untrusted party ?

Reply:
Oh it's done in a later commit